addons/website/models/website_visitor.py

```diff
@@ -79,48 +82,69 @@ def _get_visitor_hash(self, visitor_id):
         db_secret = request.env['ir.config_parameter'].sudo().get_param('database.secret')
         return hmac.new(str(visitor_id).encode('utf-8'), db_secret.encode('utf-8'), sha256).hexdigest()
```

Comment:
need to concat a uniq key to visitor id !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!